codes/models/alt_growth_rates/IRN_adm1.do

```diff
@@ -172,27 +173,34 @@ graph drop hist_irn qn_irn
 
 // predicted "actual" outcomes with real policies
 *predict y_actual if e(sample)
-predictnl y_actual = p_1*_b[p_1] + p_2* _b[p_2] + p_1_x_Tehran*_b[p_1_x_Tehran] + p_2_x_Tehran*_b[p_2_x_Tehran] ///
-+ _b[_cons] + __hdfe1__ + __hdfe2__ if e(sample), ci(lb_y_actual ub_y_actual)
+predictnl y_actual = ///
```

Comment:
We need to add testing regime in here. It is not in the treatment equation but should be in the predicted formula. I checked and it is missing in KOR too (my bad)

Reply:
fixed in all countries